static-assets/components/cstudio-dialogs-templates/approve.html

```diff
@@ -63,8 +63,9 @@ <h2 class="aprove-subtitle" data-translation="approveDialogPublishContent">Publi
         </div>
 
         <div class="form-group">
-          <h2 id="approveSubtitle" class="aprove-subtitle" data-translation="approveDialogSubmissionComment">Submission Comment</h2>
+          <h2 class="aprove-subtitle approveDialogSubmissionComment" data-translation="approveDialogSubmissionComment">Submission Comment</h2>
           <textarea class="form-control submission-comment" aria-labelledby="approveSubtitle"></textarea>
+          <p class="validation submissionCommentVal" style="display: none" id="submissionCommentVal" data-i18n="publishDialog.submissionCommentValText">/p>
```

Comment:
Check if id is necessary and otherwise remove

Reply:
Done